restapi/src/main/java/io/stargate/web/docsapi/service/ReactiveDocumentService.java

```diff
@@ -439,7 +439,6 @@ public ReactiveDocumentService(
 
   // we need to transform the stuff to support array elements
   private List<String> processSubDocumentPath(List<String> subDocumentPath) {
-    // TODO after https://github.com/stargate/stargate/pull/1105 encode chars as well
```

Comment:
I don't get this now why is the path not escaped here, are we only doing so for the fields?

Reply:
OK I think this can be resolved now, right @EricBorczuk ?